contracts/TokenWrapper.sol

```diff
@@ -69,6 +69,9 @@ contract TokenWrapper is IERC20WithCheckpointing, IForwarder, IsContract, ERC20V
      * @notice Wrap `@tokenAmount(self.outsideToken(): address, _amount)`
      * @param _amount Amount to wrap
      */
+    // review(@izqui): consider renaming 'wrap'
+    // review(@izqui): consider adding a 'to' argument that allows the depositer to decide who will receive the wrapped tokens,
```

Comment:
I'm not entirely convinced this is such a good strategy for hot / cold wallets vs. something more akin to delegation.

That being said, I'm open to including `depositTo()` and `withdrawTo()` once we hear people actually complaining / wanting them.